gdsfactory/routing/add_fiber_array.py

```diff
@@ -80,6 +82,8 @@ def add_fiber_array(
     else:
         gc = grating_coupler
     gc = gf.get_component(gc)
+    if mirror_grating_coupler:
+        gc = gf.functions.mirror(gc)
```

Comment:
I kind of hate this. This means that if I give you a grating coupler but it doesn't align with your fiber array config, you create a new component. Why? Just rotate and/or mirror the one I give